packages/manager/modules/manager-pci-common/src/api/data/image.ts

```diff
@@ -42,7 +42,25 @@ export type TImage = {
   visibility: string;
 };
 
-export const getImages = async (projectId: string): Promise<TImage[]> => {
-  const { data } = await v6.get<TImage[]>(`/cloud/project/${projectId}/image`);
+export type TGetImagesParams = {
+  flavorType?: string;
+  region?: string;
+};
+
+export const getImages = async (
+  projectId: string,
+  { flavorType, region }: TGetImagesParams,
```

Comment:
Why not giving directly params already checked by the caller? 
```suggestion
  {params }: URLSearchParams,
```